src/ui/component/filePicker/i18n/filePicker_de.json

```diff
@@ -0,0 +1,10 @@
+{
+  "file-picker": {
```

Comment:
We usually use `camelCase` instead of `kebabCase` as translations keys, it could be better to harmonize.